helm/app-operator-chart/templates/network-policy.yaml

```diff
@@ -0,0 +1,20 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: {{ .Values.name }}-network-policy
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ .Values.name }}
+  ingress:
+  - ports:
+    - port: {{ .Values.port }}
+      protocol: {{ .Values.protocol }}
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 0.0.0.0/0
```

Comment:
I'm sorry to ask, but does it basically open to all address?

Reply:
Yes.
Also, we're usually doing just `{}`, so maybe let's stick to single style